src/window/winit_window/windowed_context.rs

```diff
@@ -85,6 +85,11 @@ mod inner {
         pub fn swap_buffers(&self) -> Result<(), WindowError> {
             Ok(())
         }
+
+        /// Enables or disabled vsync.
+        pub fn set_vsync(&self, _enabled: bool) -> Result<(), WindowError> {
```

Comment:
It's a bit confusing to have this method on web when it doesn't do anything. Maybe just delete it? If you use the method it will not compile on web and you will be aware that it's not supported.

Reply:
I initially added it since the Web Context also has a few other no-op methods and it looked like adding empty stub was the way to go here.

Anyways, I pushed an update to remove the empty method :)

Reply:
Yeah, I totally see that. Maybe they should be removed as well 🤔 Anyway, thanks for the contribution 🙏